hub/developer_models_index.html

```diff
@@ -5,6 +5,7 @@
 background-class: hub-background
 body-class: hub
 summary: Get plug & play models to accelerate ML development.
+category: developer
 ---
 
-{% include hub_developer_tags_and_cards.html category="developers" %}
+{% include hub_developer_tags_and_cards.html category= developers %}
```

Comment:
The category still needs quotations around it if it's going to be used in `hub_developer_tags_and_cards`